README.md

```diff
@@ -1,7 +1,104 @@
 # Taskcluster Agile Process Tracking
 
+* [Definitions](#definitions)
+* [Tools](#tools)
+* [Roles](#roles)
+* [Sprint process](#sprint-process)
+
+___
+
 ## Definitions
 
+In order to avoid confusion when discussing our process, it's helpful to define the various terms we use to categorize our work at different levels of granularity and how they roll up into each other.
+
+We generally follow [the Atlassian model](https://www.atlassian.com/agile/project-management/epics-stories-themes), with some naming changes to avoid confusion in [our tooling](#Tools).
+
+Theme -> Initiative -> Milestone -> Epic -> Issue
+
+### Theme
+Themes are large focus areas than span the Mozilla organization and are pertinent and addressable by the Taskcluster team.
+
+For Taskcluster @ Mozilla, we are currently tracking the following themes:
+* Making Data Actionable
+* Ease of use
+* Promote external adoption
+* Cost Reduction
+
+### Initiative
+An _initiative_ is a collection of one or more milestones that, taken together, address one or more themes. Depending on the project area, an initiative can be a thin wrapper around a single milestone if that milestone is high value and self-contained.
```

Comment:
Since the definition of Initiative depends on the definition of Milestone, would it be better to define Milestone first?

Reply:
I could invert the list, but each level is aside from issue is meaningless without the rest anyway. Hopefully the examples will make it more clear.